meson.build

```diff
@@ -13,7 +13,7 @@
 #  limitations under the License.
 
 project('lc3', 'c',
-	version: '0.1',
+	version: '1.0.0',
```

Comment:
I suggest to set 1.0.1.
In this way, I can publish a new release with the right version number.

Reply:
Done